packages/performance/src/services/remote_config_service.ts

```diff
@@ -219,3 +267,30 @@ function configValid(expiry: string): boolean {
 function shouldLogAfterSampling(samplingRate: number): boolean {
   return Math.random() <= samplingRate;
 }
+
+/**
+ * True if event should be sent to Fl transport endpoint rather than log endpoint.
+ * rolloutPercent is in range [0.0, 100.0].
+ * @param {string} iid Installation ID which identifies a web app installed on client.
+ * @param {number} rolloutPercent the possibility of this app sending events to Fl endpoint.
+ * @return {boolean} true if this installation should send events to Fl endpoint.
+ */
+export function isDestFl(iid: string, rolloutPercent: number): boolean {
+  if (iid.length === 0) {
+    return false;
+  }
+  return getHashPercent(iid) < rolloutPercent;
```

Comment:
SDK will honor the percentage because we don't perform special handling if it is out of valid range, backend can inspect and control flag value if this is not correct.

Reply:
Sure, even if BE does that, having a range check would make the SDK cleaner.

Reply:
Currently we don't have range check for RC fetch result. We can add them but I don't expect SDK to handle this differently if the value is out-of-range, therefore not adding additional check for now and revisit it in the future.